src/__init__.py

```diff
@@ -0,0 +1,10 @@
+from .bacterial_parser import *
```

Comment:
I organized the code into modules for a clearer structure, but I don't understand how this is suppose to make things easier if I then I have to import everything again.

Reply:
Check about how the code looks like for the user. 
https://www.reddit.com/r/Python/comments/1bbbwk/whats_your_opinion_on_what_to_include_in_init_py/